gnomad/utils/vcf.py

```diff
@@ -1158,6 +1158,7 @@ def make_vcf_filter_dict(
 def make_hist_bin_edges_expr(
     ht: hl.Table,
     hists: List[str] = HISTS,
+    row_name: str = "",
```

Comment:
Can we be more descriptive than`row_name`? Maybe `ann_w_hists` or something? Could you also make this Optional? And  whats the benefit of defaulting to an empty string instead of None?